services/viva-ms/serverless.yml

```diff
@@ -13,10 +13,12 @@ package:
 provider:
   lambdaHashingVersion: 20201221
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
   stage: dev
   region: eu-north-1
   endpointType: regional
+  eventBridge:
```

Comment:
are you sure that `eventBridge` is a property in serverless provider? It is not in their documentation.